zipline/utils/paths.py

```diff
@@ -0,0 +1,223 @@
+"""
+Canonical path locations for zipline data.
+
+Paths are rooted at $ZIPLINE_ROOT if that environment variable is set.
+Otherwise default to expanduser(~/.zipline)
+"""
+from errno import EEXIST
+import os
+from os.path import exists, expanduser, join
+
+import pandas as pd
+
+
+def hidden(path):
+    """Check if a path is hidden.
```

Comment:
Does this need to be a cross-platform notion of hidden, e.g. using windows file attributes, or just looking at the name?